packages/eslint-config-base/index.js

```diff
@@ -1,7 +1,13 @@
 // @flow
 
 module.exports = {
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    requireConfigFile: false,
+    babelOptions: {
+      presets: ['@babel/preset-flow'],
+    },
+  },
```

Comment:
As I understand, previously, that config didn't require Flow and wasn't intended for it. I think we shouldn't rely on Flow here.